katas/chicken-sexing/mateo/chickenSexing.js

```diff
@@ -1,3 +1,14 @@
 export function correctness(bobsDecisions, expertDecisions) {
+    let points = 0;
```

Comment:
Please move these changes to a separate branch PR

Reply:
Just noticed that these changes are in https://github.com/fjcoding/AT-BC-LAT-01/pull/4/files, so for this PR, please remove changes for chickenSexing.js and chickenSexing.test.js